adoptingerlang.org

```diff
@@ -2024,13 +2071,13 @@ Now when running the release we must set these variables or the release will fai
 
 #+BEGIN_SRC shell
 $ LOGGER_LEVEL=debug SCHEDULERS=1 _build/default/rel/service_discovery/bin/service_discovery console
-Erlang/OTP 21 [erts-10.3.3] [source] [64-bit] [smp:1:1] [ds:1:1:10] [async-threads:30] [hipe]
+Erlang/OTP 21 [erts-10.4.4] [source] [64-bit] [smp:1:1] [ds:1:1:10] [async-threads:30] [hipe]
 ...
 #+END_SRC
 
 *** Coming Up
 
-To get a feel for what we will be doing in the next chapters copy =_build/prod/rel/service_discovery/service_discovery-c9e1c805d57a78d9eb18af1124962960abe38e70.tar.gz= to =/tmp/service_discovery=, unpack it and start the node. Here I am assuming that the necessary environment variables are already exported and not repeating them on the line running the bin script's commands.
+To get a feel for what we will be doing in the next chapters, copy =_build/prod/rel/service_discovery/service_discovery-c9e1c805d57a78d9eb18af1124962960abe38e70.tar.gz= to =/tmp/service_discovery=, unpack it and start the node. Here I am assuming that the necessary environment variables are already exported and not repeating them on the line running the bin script's commands.
```

Comment:
How would someone check if they have the right environment variables defined?

Reply:
Yea, I'll just add the exports.